### PR TITLE
Best practices: remove return 🎨

### PR DESCRIPTION
**Pull request by RealHinome**

### CODE REVIEW

File: src/helpers.rs [hunk 1]
```diff
@@ -93,7 +93,7 @@
         Ok(d) => {
             encode(&Header::new(Algorithm::RS256), &Claims {
                 sub: user_id.to_lowercase(),
-                aud: if finger.is_some() { Some(finger.unwrap()[0..24].to_string()) } else { None },
+                aud: if let Some(val) = finger { Some(val[0..24].to_string()) } else { None },
```
[the remark this answers is not in the document]
Comment (n1c00o) [reply src/helpers.rs:96]: ^

File: src/helpers.rs [hunk 2]
```diff
@@ -119,9 +119,9 @@ pub fn get_jwt(token: String) -> Result<TokenData<Claims>, String> {
 }
 
 pub fn get_age(year: i32, month: u32, day: u32) -> f64 {
-    return (((SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis()
+    (((SystemTime::now().duration_since(UNIX_EPOCH).unwrap().as_millis()
```
Comment (n1c00o) [inline src/helpers.rs:122]: Unwrap ?

Comment (RealHinome) [reply src/helpers.rs:123]: I assume you mean I have to handle the unwrap?

File: src/database/cassandra.rs [hunk 3]
```diff
@@ -32,6 +32,7 @@ pub async fn query(query: &'static str, params: Vec<String>) -> scylla::QueryRes
     SESSION.get().unwrap().query(query, params).await.expect("Query error")
 }
 
+#[allow(clippy::type_complexity)]
 pub async fn update_user(params: (String, Option<String>, Option<String>, Option<String>, Option<String>, String, String)) -> scylla::QueryResult {
     SESSION.get().unwrap().query("UPDATE accounts.users SET username = ?, avatar = ?, bio = ?, birthdate = ?, phone = ?, email = ? WHERE vanity = ?", params).await.expect("Query error")
```
Comment (n1c00o) [inline src/database/cassandra.rs:37]: `expect` causes panic, this should be handled correctly

Comment (n1c00o) [reply src/database/cassandra.rs:37]: Same for `unwrap`

File: src/helpers.rs [hunk 4]
```diff
@@ -38,7 +38,7 @@ pub fn init() {
 
 pub fn random_string() -> String {
```
Comment (n1c00o) [inline src/helpers.rs:39]: Functions should be documented

File: src/helpers.rs [hunk 5]
```diff
@@ -73,6 +73,7 @@ pub fn hash_test(hash: &str, pwd: &[u8]) -> bool {
     argon2::verify_encoded_ext(hash, pwd, ARGON_SECRET.get().unwrap().as_bytes(), &[]).unwrap()
 }
 
+#[allow(clippy::type_complexity)]
 pub fn encrypt(data: &[u8]) -> String {
```
Comment (n1c00o) [inline src/helpers.rs:77]: Functions should be documented

File: src/helpers.rs [hunk 6]
```diff
@@ -119,9 +120,14 @@ pub fn get_jwt(token: String) -> Result<TokenData<Claims>, String> {
 }
 
 pub fn get_age(year: i32, month: u32, day: u32) -> f64 {
```
Comment (n1c00o) [inline src/helpers.rs:122]: Functions should be documented

File: src/database/cassandra.rs (hunk 3 above)
Comment (RealHinome) [reply src/database/cassandra.rs:37]: If no session is in progress, I prefer it to crash. Because anyway, you won't be able to do anything without

**Review by n1c00o — APPROVED** [review]
lgtm at the moment

**Pull request merged**